code/controllers/subsystem/vote.dm

```diff
@@ -131,59 +138,64 @@ SUBSYSTEM_DEF(vote)
 					restart = TRUE
 			if("gamemode")
 				if(GLOB.master_mode != .)
-					SSticker.save_mode(.)
-					if(SSticker.HasRoundStarted())
-						restart = TRUE
-					else
+					if(!SSticker.HasRoundStarted())
 						GLOB.master_mode = .
 			if("map")
 				SSmapping.changemap(global.config.maplist[.])
 				SSmapping.map_voted = TRUE
+			// if("transfer")
+			// 	if(. == "Initiate Crew Transfer")
+			// 		SSshuttle.requestEvac(null, "Crew Transfer Requested.")
+			// 		var/obj/machinery/computer/communications/C = locate() in GLOB.machines
+			// 		if(C)
+			// 			C.post_status("shuttle")
 	if(restart)
 		var/active_admins = FALSE
-		for(var/client/C in GLOB.admins)
+		for(var/client/C in GLOB.admins+GLOB.deadmins)
 			if(!C.is_afk() && check_rights_for(C, R_SERVER))
 				active_admins = TRUE
 				break
 		if(!active_admins)
-			SSticker.Reboot("Restart vote successful.", "restart vote", 1) //no delay in case the restart is due to lag
+			SSticker.Reboot("Restart vote successful.", "restart vote")
 		else
 			to_chat(world, "<span style='boldannounce'>Notice:Restart vote will not restart the server automatically because there are active admins on.</span>")
 			message_admins("A restart vote has passed, but there are active admins on with +server, so it has been canceled. If you wish, you may restart the server.")
 
 	return .
 
 /datum/controller/subsystem/vote/proc/submit_vote(vote)
-	if(mode)
-		if(CONFIG_GET(flag/no_dead_vote) && usr.stat == DEAD && !usr.client.holder)
-			return FALSE
-		if(!(usr.ckey in voted))
-			if(vote && 1<=vote && vote<=choices.len)
-				voted += usr.ckey
-				choices[choices[vote]]++ //check this
-				return vote
-	return FALSE
-
-/datum/controller/subsystem/vote/proc/initiate_vote(vote_type, initiator_key)
-	if(!Master.current_runlevel) //Server is still intializing.
-		to_chat(usr, "<span class='warning'>Cannot start vote, server is not done initializing.</span>")
+	if(!mode)
+		return FALSE
+	if(CONFIG_GET(flag/no_dead_vote) && (usr.stat == DEAD && !isnewplayer(usr)) && !usr.client.holder && mode != "map")
+		return FALSE
```

Comment:
Why the extra code here?

Reply:
NOT sure about this one, not my change, I will revert this